### PR TITLE
Document how to test changes in sjb/ literal content

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,11 @@ sjb/push-update.sh sjb/generated/YOUR_TEST_JOB.xml
 ````
 Cleanup of these jobs post-test is still manual.
 
+If changes are being made to the files under `sjb/` in this repository, it is not enough to copy a job configuration and run it to
+test the changes. Instead, it will be necessary to mark the copied job as syncing a pull request for `aos-cd-jobs` using the `type`
+field on the repository as per [the spec](./sjb/syntax.md#sync_repos). Then, when running your copied job, configure it at run-time
+to merge in your pull request.
```
Comment (kargakis) [inline README.md]: An example would be nice - testing in sjb probably deserves its own doc.

Comment (stevekuznetsov) [inline README.md]: What sort of example would be useful here?

Comment (kargakis) [inline README.md]: It's not clear what "at run-time" means. Most will use the Jenkins UI so calling that out may be enough. Also including a stanza with the exact change that needs to be included in the test job would be nice.

Comment (stevekuznetsov) [inline README.md]: I'll call out the Jenkins UI.

> Also including a stanza with the exact change that needs to be included in the test job

This is going to differ based on how you are configuring your synced repos, so I hope it is OK to assume the dev can follow the link to the syntax doc here.

**Pull request merged**